packages/@lwc/jest-serializer/src/index.js

```diff
@@ -9,8 +9,9 @@ const DOMElement = PrettyFormat.plugins.DOMElement;
 
 const cleanElementAttributes = require('./clean-element-attrs');
 
-function test({ nodeType } = {}) {
-    return (
+function test(node) {
```

Comment:
Let's make this method a little more comprehensive, by checking first if the passed argument is an object and then let's make sure if the object is a DOM node that can be serialized.

```suggestion
function test(obj) {
  if (typeof obj !== 'object' || obj === null) {
    return false;
  }
  
  const { nodeType } = obj;
  return (
        nodeType === 1 || // element
        nodeType === 3 || // text
        nodeType === 6    // comment
    );
}
```

Reply:
It make senses